src/services/useBoundingBox/index.ts

```diff
@@ -52,8 +53,25 @@ function roundBoundingBox(
   }
 }
 
-export function boundingBoxToString(boundingBox: BoundingBox): string {
-  return `${boundingBox.lonMin}°E ${boundingBox.latMin}°N, ${boundingBox.lonMax}°E ${boundingBox.latMax}°N`
+export function boundingBoxToString(
+  boundingBox: BoundingBox,
+  longitudeStepSize: number,
+  latitudeStepSize: number,
+): string {
+  const asPrecision = (value: number, step: number): string => {
+    const nIntegers = Math.floor(Math.log10(Math.abs(value)))
+    const stepStrs = step !== undefined ? step.toString().split('.') : '1'
```

Comment:
since asPrecision only excepts a `:number` step can not be undefined in this case